utility/environment_check.py

```diff
@@ -140,18 +140,28 @@ def get_status_after_execution():
     leftovers = {'Leftovers added': [], 'Leftovers removed': []}
     for kind, kind_diff in diffs_dict.items():
         if ADDED_RESOURCE in kind_diff:
-            leftovers['Leftovers added'].append({
-                kind: kind_diff[ADDED_RESOURCE][
-                    ''.join(kind_diff[ADDED_RESOURCE])
-                ]
-            })
+            try:
+                leftovers['Leftovers added'].append({
```

Comment:
@RazTamir Do you have an example of what this ends up looking like once items are appended? The logic here around what we're appending isn't super clear. I know you haven't changed it here in this PR but I'm just curious if we could simplify it at all.

Reply:
sure:
http://pastebin.test.redhat.com/774998
In that dictionary, you have a key named 'iterable_item_added' or 'iterable_item_removed' that includes a dict or list of more dictionaries that represent the added/removed items